### DISCUSSION TITLE
馬克士威方程組真偉大！

### DISCUSSION THREAD

**Discussion by LeoTsai999** [1101-GS4719-C/Python-Community-GS4719-C#2139] [General]
馬克士威提出了電磁方程組，預測電磁波的存在。我覺得這是一個非常重要的貢獻。
這件事啟發了我對於理論研究的想法。我認為一個理論物理學家，應該要像馬克士威一樣從理論中預測某些目前實驗上還未證實的事情，給予科學研究一個方向！

**Comment by seastar0687** [comment]
馬克士威方程組完美地描述了電磁場之間的關係，而這都要歸功於馬克士威十分深厚的數學功底，可見數學能力對於物理研究乃至於可學研究都是十分重要。
通過馬克士威方程組可以預測電磁波，或許上帝在創造世界時就是說出了馬克士威方程組才有了光呢！
